spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/GcpHeaders.java

```diff
@@ -26,4 +26,6 @@
 	private static final String PREFIX = "gcp_";
 
 	public static final String ACKNOWLEDGEMENT = PREFIX + "pub_sub_acknowledgement";
+
+	public static final String DESTINATION = PREFIX + "destination_topic";
```

Comment:
I would just call this constant (and its value)  as `topic`:
```
public static final String TOPIC = PREFIX + "topic";
```

Seems for me the name of the class is too broad as well - `GcpPubSubHeaders` would sound better.
The reason that in other modules we may have a different set of headers and, therefore, there will be a new support class for headers constants. BTW the ``prefix should might be as `gcp_pubsub_` as well. But this can be done as a separate issue anyway.

Right, I'd prefer to see the constant and its value renamed.